conda_smithy/ci_register.py

```diff
@@ -178,48 +178,41 @@ def add_project_to_travis(user, project):
     headers = travis_headers()
     endpoint = 'https://api.travis-ci.org'
 
-    url = '{}/hooks'.format(endpoint)
+    url = '{}/repos/{user}/{project}'.format(endpoint, user=user, project=project)
 
-    found = False
     count = 0
 
-    while not found:
+    while True:
         count += 1
 
         response = requests.get(url, headers=headers)
         try:
-            response.raise_for_status()
             content = response.json()
-        except (requests.HTTPError, ValueError):
```

Comment:
Sorry, was overlooking this before. We should keep the `raise_for_status` and keep the `requests.HTTPError`. We can drop the `ValueError`. This will correctly catch the non-existent repo case.